On musl, the generic support works fine.

This prevents spurious memcheck and cachegrind failures on ppc64.
--- valgrind-3.19.0/coregrind/m_syswrap/syswrap-ppc64-linux.c.old	2022-06-11 03:12:37.140000000 -0500
+++ valgrind-3.19.0/coregrind/m_syswrap/syswrap-ppc64-linux.c	2022-06-11 03:13:33.550000000 -0500
@@ -852,7 +852,7 @@
    GENX_(__NR_madvise,           sys_madvise),            // 205
 // _____(__NR_mincore,           sys_mincore),            // 206
    LINX_(__NR_gettid,            sys_gettid),             // 207
-// _____(__NR_tkill,             sys_tkill),              // 208
+   LINXY(__NR_tkill,             sys_tkill),              // 208
    LINX_(__NR_setxattr,          sys_setxattr),           // 209
 
    LINX_(__NR_lsetxattr,         sys_lsetxattr),          // 210
